docs/modules/ROOT/pages/explanations/decisions/deployments.adoc

```diff
@@ -0,0 +1,66 @@
+= Decision How to Deploy Compositions and XRDs
+
+== Problem
+
+We need to deploy our XRDs and Compositions for AppCat to OpenShift and Kubernetes clusters.
+Also writing the compositions and XRDs plainly is somewhat a hassle (yaml engineering).
+Having helpers with engineering compositions and XRDs would be a welcomed bonus for the solution.
+
+== Requirements
+
+* Manage XRDs
+* Manage Compositions
+* Manage Providers and their config
+* Promote re-usability were possible
+* Help with developing XRDs and Compositions
+* Should support Crossplane packages in the future
+* Should support APPUiO Manage in the future
+
+== Proposals
+
+=== Proposal 1: Fine Grained Commodore Packages
+
+This proposal will enhance `+component-crossplane+` in such a way that it can also deploy XRDs and compositions via parameters.
```

Comment:
```suggestion
This proposal will enhance https://github.com/projectsyn/component-crossplane[`+component-crossplane+`] in such a way that it can also deploy XRDs and compositions via parameters.
```